vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -1164,6 +1164,69 @@ void NativeUIManager::measure(
   callback(args);
 }
 
+void NativeUIManager::findSubviewIn(
+    facebook::react::ShadowNode &shadowNode,
+    float x,
+    float y,
+    facebook::xplat::module::CxxModule::Callback callback) {
+  ShadowNodeBase &node = static_cast<ShadowNodeBase &>(shadowNode);
+  auto view = node.GetView();
+
+  auto rootUIView = view.as<winrt::UIElement>();
```

Comment:
this needs to be try_as if you want to have null checks/think it might not be one, .as will just throw

Reply:
OK.